api/command_msg_test.go

```diff
@@ -0,0 +1,51 @@
+// Copyright (c) 2016 Mattermost, Inc. All Rights Reserved.
+// See License.txt for license information.
+
+package api
+
+import (
+	"strings"
+	"testing"
+
+	"github.com/mattermost/platform/model"
+	"github.com/mattermost/platform/store"
+)
+
+func TestMsgCommands(t *testing.T) {
+	Setup()
+
+	team := &model.Team{DisplayName: "Name", Name: "z-z-" + model.NewId() + "a", Email: "test@nowhere.com", Type: model.TEAM_OPEN}
```

Comment:
Can we switch to `success+test@simulator.amazonses.com` or `success+blahblah@simulator.amazonses.com` for email.  We sometimes run unit tests with email enabled on our test servers and because of the hard bounces we get blocked as a spammer from amazon.